tests/specs/openai/conventional-commits.ts

```diff
@@ -0,0 +1,161 @@
+import { readFile } from 'fs/promises';
+// eslint-disable-next-line unicorn/import-style
+import path, { dirname } from 'path';
```

Comment:
Remove the ESLint disable, and use `dirname` via `path.dirname`

Reply:
Fixed